context/app/static/js/components/detailPage/entityHeader/EntityHeaderContent/EntityHeaderContent.jsx

```diff
@@ -54,9 +55,11 @@ function EntityHeaderContent({ assayMetadata, shouldDisplayHeader, vizIsFullscre
             <>
               <StyledSvgIcon component={entityIconMap[entity_type]} />
               <EntityHeaderItem text={hubmap_id} />
-              {Object.entries(entityToFieldsMap[entity_type]).map(([label, fn]) => (
-                <EntityHeaderItem text={fn(assayMetadata) || `undefined ${label}`} key={label} />
-              ))}
+              {entity_type in entityToFieldsMap
+                ? Object.entries(entityToFieldsMap[entity_type]).map(([label, fn]) => (
+                    <EntityHeaderItem text={fn(assayMetadata) || `undefined ${label}`} key={label} />
+                  ))
+                : `No fields defined for entity type ${entity_type}`}
```

Comment:
I would prefer not to show anything for support entities.

Reply:
I agree! I've added an empty object as the value for the `Support` key in `entityToFieldsMap` above on line 32. Since `Object.keys({})` returns an empty array, no items are displayed in the header for support entities. I also added the `else` case here to be type safe, since `Object.keys(undefined)` throws `TypeError: can't convert undefined to object`, which is currently causing Support pages on prod to crash to a white screen if you scroll down, e.g.: https://portal.hubmapconsortium.org/browse/support/d04a1abdf3546caf1896f0d9923027db

I'll add a comment above the `Support: {}` to indicate we are explicitly not showing anything for this entity.

Reply:
You could also return null or use a short-circuit, right?